src/server/server.js

```diff
@@ -31,6 +32,13 @@ io.sockets.on('connection', socket => {
   });
 
   socket.on('createRoom', () => {
+    if(socket.room) {
+      socket.room.disconnect(socket);
+      if (socket.room.players.length === 0) {
+        socket.room.close();
+        rooms.delete(socket.room.id);
+      }
+    }
```

Comment:
This fixes a bug where a client creating multiple rooms would have the same game start multiple times on their screen when another player joined the room.